### DISCUSSION TITLE
Remove or replace Twitter/X

### DISCUSSION THREAD

**Discussion by svoop** [rubygems/rubygems.org#4277] [General]
I've just pushed a gem release and got the email confirming it. In the footer:

<img width="621" alt="image" src="https://github.com/rubygems/rubygems.org/assets/7255/5e35b802-e24d-4d38-ac27-043821c87d6a">

Now, put aside the new management and tone of Twitter/X for a moment, it's not possible to view the Rubygems Twitter/X profile without having logged in to Twitter/X. This renders this microblog useless for folks like me who – for one reason or another – have decided to leave Twitter/X.

Obviously, I can't see whether there's anything interesting happening on the Rubygems Twitter/X profile, so I can't tell which of the following actions would be appropriate:

* Drop Twitter/X without replacement.
* Drop Twitter/X in favor of Mastodon – there's already a (not very active) profile there on https://ruby.social/@rubygems

What do you think?

**Comment by simi** [comment]
Hello @svoop and thanks for the report. I do agree the fact link doesn't show content to anyone is actually problem. Until now I had no idea about the google group as well. I think we can provide better set of links including https://blog.rubygems.org/. I'll check what's status of https://ruby.social/@rubygems as well.

**Comment by svoop** [comment]
Seems, this has been changed at some point, thanks @simi !!

**Reply by simi** [reply]
was removed at https://github.com/rubygems/rubygems.org/pull/4695
